packages/react-admin/src/mui/detail/Create.js

```diff
@@ -156,10 +162,6 @@ Create.propTypes = {
     hasList: PropTypes.bool,
 };
 
-Create.defaultProps = {
```

Comment:
Why did you remove that?

Reply:
Because it's not used anywhere in this component, unlike in the `Edit` component